.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+- [ ] Closes #xxxx (Replace xxxx with the GitHub issue number)
```

Comment:
The "Replace xxxx ..." comment can be an HTML comment, so it doesn't show up in the PR description if the OP doesn't remove it.